app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] You did a great job by adding devise to the user model as required, but I have noticed that you have missed one important requirement for devise. An email should be confirmed before an account is fully registered, kindly add one devise attribute which will enable __email confirmation__.

![confirm](https://user-images.githubusercontent.com/53124548/226846569-39f05b05-d3ef-437f-8da7-948267a80623.png)